extensions/amp-story-interactive/0.1/amp-story-interactive-img.css

```diff
@@ -58,11 +59,35 @@
 .i-amphtml-story-interactive-img-option-img {
   border-radius: 50% !important;
   overflow: hidden !important;
-  position: absolute !important;
-  top: 0 !important;
-  left: 0 !important;
   width: 100% !important;
   height: 100% !important;
   background-position: center !important;
   background-size: cover !important;
 }
+
+.i-amphtml-story-interactive-option-selected
+  .i-amphtml-story-interactive-img-option-img {
+  box-shadow: inset 0 0 0 0.25em var(--i-amphtml-interactive-option-post-color) !important;
+}
+
+.i-amphtml-story-interactive-post-selection.i-amphtml-story-interactive-has-data
+  .i-amphtml-story-interactive-img-option-img::before {
+  background-color: var(--i-amphtml-interactive-option-post-color) !important;
+  content: '' !important;
+  display: block !important;
+  width: 100% !important;
+  height: 100% !important;
+  margin-top: calc(100% - var(--option-percentage)) !important;
+  opacity: 0.5 !important;
+}
+
+.i-amphtml-story-interactive-post-selection.i-amphtml-story-interactive-has-data
+  .i-amphtml-story-interactive-img-option-percentage-text {
+  position: absolute !important;
+  bottom: 0.625em !important;
+  text-align: center !important;
+  right: 0 !important;
+  left: 0 !important;
+  color: white !important;
+  font-weight: bold !important;
```

Comment:
We typically use a numeric value for this so the browser and reader of the code can know exactly what weight is being used:
https://github.com/ampproject/amphtml/blob/main/extensions/amp-story-interactive/0.1/amp-story-interactive-binary-poll.css#L120